### PR TITLE
Describe role of Working Group co-chairs. 

### PR DESCRIPTION
**Pull request by cdavisafc**
I have also made provisional changes to the governance to reflect the name change as discussed in #64 and PR #84.

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -89,6 +89,19 @@ The aspiration is no one company or organization should have oversight of the ov
...
 
+### Working Group Co-Chairs
+
+The GitOps Working Group draws together individuals who are collaborating on forming content and technology "to provide companies and individuals with the skills, knowledge and competency to implement GitOps tooling and methodologies which simplify the operation and management of infrastructure and cloud native applications" and to care for the OpenGitOps  project.
+
+Co-chairs:
+
+- Ensure that meetings and other activities are conducted and progress continues to be made against the project agenda
+- Have GitOps domain knowledge
+- Will engage deeply in the work being done by the working group - that is, the role is not purely administrative. They are, however not responsible for performing all of the work - this is shared across the working group members and expected from those who have volunteered onto various workstreams.
```
Comment (sbose78) [inline GOVERNANCE.md:100]: Wondering if we should use the term "technical coordination" somewhere to emphasize that this isn't just an administrative role ?

**Review by sbose78 — APPROVED** [review]
Looks good to me, had a couple of comments :)

**Comment by scottrigby** [comment]
Also see https://github.com/gitops-working-group/gitops-working-group/discussions/40

File: GOVERNANCE.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ Maintainers will be invited to the `@gitops-working-group/maintainers` <https://
 
 ### Oversight Committee
 
-This committee is responsible for the overall project, and anything not easily managed by the Maintainers of each git repository. Including:
+The OpenGitOps oversight committee is responsible for the overall project, and anything not easily managed by the Maintainers of each git repository. Including:
```
[the remark this answers is not in the document]
Comment (scottrigby) [reply GOVERNANCE.md:73]: There is currently no oversight committee. This was a provisional governance draft for the project prior to rename. The current idea is to simplify this part "out". See https://github.com/gitops-working-group/gitops-working-group/pull/86#issuecomment-806104810

File: GOVERNANCE.md [hunk 3]
```diff
@@ -89,6 +89,19 @@ The aspiration is no one company or organization should have oversight of the ov
...
+### Working Group Co-Chairs
+
+The GitOps Working Group draws together individuals who are collaborating on forming content and technology "to provide companies and individuals with the skills, knowledge and competency to implement GitOps tooling and methodologies which simplify the operation and management of infrastructure and cloud native applications" and to care for the OpenGitOps  project.
+
+Co-chairs:
+
+- Ensure that meetings and other activities are conducted and progress continues to be made against the project agenda
+- Have GitOps domain knowledge
+- Will engage deeply in the work being done by the working group - that is, the role is not purely administrative. They are, however not responsible for performing all of the work - this is shared across the working group members and expected from those who have volunteered onto various workstreams.
+- When necessary, will coordinate across different working group workstreams.
```
Comment (scottrigby) [inline GOVERNANCE.md:101]:
```suggestion
- When necessary, will handle human and technical coordination across different working group workstreams.
```

**Review by scottrigby — APPROVED** [review]
See https://github.com/gitops-working-group/gitops-working-group/pull/86#issuecomment-806104810 for context. Further PRs coming.

**Pull request merged**